frontend/server/src/Controllers/Contest.php

```diff
@@ -754,6 +754,46 @@ public static function getContestDetailsForTypeScript(
             $result['smartyProperties']['title'] = new \OmegaUp\TranslationString(
                 'omegaupTitleContest'
             );
+
+            if (!is_null($contest->rerun_id) && $contest->rerun_id !== 0) {
```

Comment:
nit:

```suggestion
            if (!empty($contest->rerun_id)) {
```

es un poquitín más corto. y también, esto jamás debería ser cero (algo sobre "valores mágicos").

se puede crear otro cambio para agregarle un `FOREIGN KEY CONSTRAINT`? para poder hacer eso, habría que cambiar todos los ceros por `NULL`s.

Reply:
Listo, ya apliqué el cambio de la base en el PR #5606